presentation/src/main/java/me/androidbox/presentation/event/screen/EventScreen.kt

```diff
@@ -44,6 +51,8 @@ fun EventScreen(
     modifier: Modifier = Modifier) {
 
     val calendarStateTimeDate = rememberUseCaseState()
+    val bottomSheetState = rememberModalBottomSheetState()
+    val bottomSheetScope = rememberCoroutineScope()
```

Comment:
As these are related to the UI I am declaring them there for the bottomSheetState and bottomSheetState. Would this be the best place to have them declared here?

Reply:
Personally, since you will probably reuse the bottomsheet here, and in the reminder screen and task screen, you can have it declared in the AgendaBottomSheet itself